WMFComponents/Sources/WMFComponents/Components/Activity Tab/WMFActivityTabViewModel.swift

```diff
@@ -11,6 +11,7 @@ public class WMFActivityTabViewModel: ObservableObject {
     @Published var minutesRead: Int = 0
     @Published var totalArticlesRead: Int = 0
     @Published var dateTimeLastRead: String? = nil
+    @Published var weeklyReads: [Int]? = []
```

Comment:
Minor suggestion to bundle these module properties into a struct in case it helps with performance (see previous PR comment about this).